sdk/digitaltwins/azure-digitaltwins-core/autorest.md

```diff
@@ -40,4 +40,5 @@ java:
     context-client-method-parameter: true
     custom-types-subpackage: models
     required-fields-as-ctor-args: true
+    custom-types: DigitalTwinModelsAddOptions,DigitalTwinModelsDeleteOptions,DigitalTwinModelsGetByIdOptions,DigitalTwinModelsUpdateOptions,DigitalTwinsAddOptions,DigitalTwinsAddRelationshipOptions,DigitalTwinsDeleteOptions,DigitalTwinsDeleteRelationshipOptions,DigitalTwinsGetByIdOptions,DigitalTwinsGetComponentOptions,DigitalTwinsGetRelationshipByIdOptions,DigitalTwinsListIncomingRelationshipsOptions,DigitalTwinsListRelationshipsOptions,DigitalTwinsUpdateComponentOptions,DigitalTwinsUpdateOptions,DigitalTwinsUpdateRelationshipOptions,EventRoutesAddOptions,EventRoutesDeleteOptions,EventRoutesGetByIdOptions,EventRoutesListOptions,QueryTwinsOptions
```

Comment:
By declaring these as custom types, they are moved to the appropriate namespace so that we can use them in our convenience layer APIs. Note that some options classes aren't here on purpose since we have to create wrapper classes for some of them instead in order to add more options to them